### ISSUE TITLE
Training Error : AttributeError: 'InstructBlipConfig' object has no attribute 'hidden_size'

### ISSUE THREAD

**Issue by AndyDu0116** [HaozheZhao/MIC#5]
尝试训练instructblip-flan-t5-xxl时遇到了**AttributeError: 'InstructBlipConfig' object has no attribute 'hidden_size'**，请问您该如何解决？
![image](https://github.com/HaozheZhao/MIC/assets/66242411/b45cfb1d-072d-47d3-b568-49fc11c29f7b)
![image](https://github.com/HaozheZhao/MIC/assets/66242411/dc0f3797-5cdc-4fad-9c4d-fe6e0ec1a3b7)

**Comment by HaozheZhao** [comment]
Hi there, regarding your question about the problem during the training of MMICL, it seems that running the code from the repository is working fine. 
So, it would be helpful if you could provide more information about the environment I have used, and also where you imported the 'InstructBlipConfig' and how you initialized it.

**Comment by AndyDu0116** [comment]
非常感谢您的回复！
1.我的实验环境是：python==3.8.5; deepspeed==0.7.4;transformers==4.26.1;torch1.13.1+cu117；环境配置和项目requirements.txt保持一致.
2.InstructBlipConfig是在 /task/vqa/get_trainer.py文件中第31行 config = InstructBlipConfig.from_pretrained(model_args.model_name_or_path)初始化的。进一步，InstructBlipConfig类是在/model/instructblip/configuration_instructblip.py脚本中定义的，代码中InstructBlipConfig类定义中没有hidden_size属性，这是是否导致了上文截图中最后的报错呢。
希望您能给一些解决建议，感谢！

**Comment by HaozheZhao** [comment]
Sorry about that, I believe this problem is due to the version of transformers. Specifically, right after the release of transformer 4.26.1, there was a pull request to fix a similar issue in deepspeed. This fix has been merged into the 4.27.0 version of transformers. It adds a check to only determine hidden size if necessary.

If you are interested, you can reference this pull request: [pull](https://github.com/huggingface/transformers/pull/21504).

In short, you can simply update your transformers to a new version, such as 4.28.1.

I'm deeply sorry for my mistake, forgetting to check the transformers version in requirements.txt. I have update the requirements and provide a requirements.yml. You can just reload the environment in the following commands:
```
conda env create -f environment.yml
```

**Comment by AndyDu0116** [comment]
好的，已经可以跑起来了，非常感谢您的回复！

**Comment by HaozheZhao** [comment]
That's great! 😃 
I am sorry again, but currently this project is facing a shortage of staff (<=1).
The enhanced version of the code, the research paper, and the detailed data preprocessing will be published in the next two to three days.
